lib/rules/no-identical-tests.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview disallow identical tests
+ * @author 薛定谔的猫<hh_2013@foxmail.com>
+ */
+
+'use strict';
+
+const utils = require('../utils');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'disallow identical tests',
+      category: 'Tests',
+      recommended: true,
```

Comment:
Can you set this to `false` for now? I agree that it would be good to set it to `true` eventually, but that will be a breaking change, so I think it would be better to set it to `false` at first and then change it a bit afterwards.